### ISSUE TITLE
Add a CI job for running tests on Browserstack with iOS

### ISSUE THREAD

**Issue by robert-altom** [alttester/AltTester-Unity-SDK#1316]

**Comment by sandraaltom** [comment]
Triaged Week #38, 2023.

**Comment by sandraaltom** [comment]
We need this ticket before starting to test the WebSocket part.

**Comment by MihaiCuc98** [comment]
What was implemented:
- updated `push.yml` to:
    - build .ipa file
    - run python tests on .ipa file using Browserstack
- updated `conftest.py` to run tests on iOS
- updated `IosBuildFromCommandLine` with new iOS app identifier `sampleGameiOS`
- renamed `webgl.yml` to `runs-at-night`:
    - changed the schedule to run at the end of every day
    - added iOS browserstack and Android browserstack jobs

**Comment by iulianatodoran** [comment]
@robert-altom will update the workflow with the host and port for the 35 runner for all the jobs and trigger the pipeline again to check for successful connection of the tests (as agreed on 30.01.24 daily status). Until then the ticket remains in Testing column.

**Comment by iulianatodoran** [comment]
Waiting for https://github.com/alttester/AltTester-Unity-SDK/pull/1494 to be merged into dev to test this ticket

**Comment by iulianatodoran** [comment]
## Testing environment
- GitHub Actions - workflow `runs-at-night`
- las changes from commit: https://github.com/alttester/AltTester-Unity-SDK/pull/1494 

## Tests performed
Checked on pipelines that:
- ✅ iOS, Android and WebGL jobs can be run with connection with the AltServer on the correct Host and Port
- ✅ the `.ipa` file is uploaded
- ✅ The Python tests run on the created `.ipa` using Browserstack
- the workflow name is changed to `runs-at-night` and runs only at night

## Testing notes
- even if some jobs still fail because of some failing tests on WebGL, the scope of this ticket was achieved 

## Conclusion
Considering this information, **I will close this ticket and move it into `Done`**.
